doc/fips.md

```diff
@@ -0,0 +1,132 @@
+# FIPS support
+
+libspdm 3.0.0 starts adding [FIPS 140-3](https://csrc.nist.gov/publications/detail/fips/140/3/final) support for [Cryptographic Algorithm Validation Program (CAVP)](https://csrc.nist.gov/Projects/cryptographic-algorithm-validation-program). The FIPS boundary is libspdm library plus crypto library.
```

Comment:
CAVP is an "offline" test program - NIST provides vectors and vendor provides answers for NIST to check.  This can be done as part of validation, and no specific implementation required.

KAT and PCT and mode output and version output are all for CMVP.

Suggest removing "for CAVP".

Reply:
done